README.md

```diff
@@ -1,2 +1,27 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+likeliness = 0
+count = 0
+
+print 'please enter your word'
+read word
+character = char{n}
+compare char {0} # compares the first character of the input with the solution
+
+if char{0} = true then :
+  likeliness = likeliness + 1
+  count = count + 1
+else : 
+  Break
+  
+while character >= count then:
+  comp {0 + n}
```

Comment:
It looks like you're using `count` as a loop counter here, so shouldn't you be comparing the `count`th charater rather than the `0 + n`th?